subproviders/cache.js

```diff
@@ -1,5 +1,5 @@
 const ProviderSubprovider = require('./json-rpc-engine-middleware')
-const createBlockCacheMiddleware = require('eth-json-rpc-middleware/block-cache')
+const { createBlockCacheMiddleware } = require('eth-json-rpc-middleware/dist/block-cache')
```

Comment:
Accessing exports beyond the main file are no longer recommended. You should be able to say:

``` suggestion
const { createBlockCacheMiddleware } = require('eth-json-rpc-middleware')
```